### DISCUSSION TITLE
Is "error on processing request" something that I should be concerned about?

### DISCUSSION THREAD

**Discussion by ikarisan** [0xERR0R/blocky#297] [Q&A]
Hi!
I am using blocky 0.16 as docker container and every time I start up the container I am getting log entries like:

[2021-09-28 17:32:52] ERROR server: error on processing request: resolution was not successful, used resolvers: 'upstream 'tcp+udp:tls:53'' and 'upstream 'tcp+udp:[2606:4700:4700::1001]:53'' errors: [dial udp [2606:4700:4700::1001]:53: connect: cannot assign requested address dial udp: lookup tls on 192.168.1.1:53: no such host]

Is this something I have to concern about?
The most interesting thing is that I don't know where 192.168.1.1 comes from. This is the IP of my router, defined as default gateway.

**Comment by 0xERR0R** [comment]
Hi,

Can you share your config.xml please?

**Reply by ikarisan** [reply]
Sure, but I have let out all my CustomDNS/mapping entries.
[config.yml.txt](https://github.com/0xERR0R/blocky/files/7246443/config.yml.txt)

**Comment by 0xERR0R** [comment]
Your  config looks good. I get the same error if I try to perform a DNS query: `error on processing request: resolution was not successful, used resolvers: 'upstream 'tcp6+udp:[2620:fe::fe]:53'' and 'upstream 'tcp+udp:[2606:4700:4700::1111]:53'' errors: [read udp [2001:db8:1::242:ac11:2]:45357->[2620:fe::fe]:53: i/o timeout read udp [2001:db8:1::242:ac11:2]:39354->[2606:4700:4700::1111]:53: i/o timeout]`

I can't also use this IPv6 resolvers with `dig -6 @2001:db8:1::242:ac11:2`, but this could be a network issue

Can you run `dig` with any IPv6 resolver?

BTW: I would recommend you to configure the "conditionalDNS" entry in order to be able to reach other devices in your network by ".fritz.box" domain (e.g. device.fritz.box):

```yaml
conditional:
  mapping:
    fritz.box: udp:192.168.1.1
```

**Reply by ikarisan** [reply]
Your resolver 2001:db8:1::242:ac11:2 is not working for me, too. 
But all my configured IPv6 servers are working fine with "dig -6 @<IPv6>"

My local domain is not fritz.box. 
These two entries are only used to resolve the two Fritz!Box hosts.
